test/e2e/test/version.go

```diff
@@ -33,11 +34,11 @@ func SkipInvalidUpgrade(t *testing.T, srcVersion string, dstVersion string) {
 func isValidUpgrade(from string, to string) (bool, error) {
 	srcVer, err := version.Parse(from)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("failed to parse version '%s': %w", from, err)
 	}
 	dstVer, err := version.Parse(to)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("failed to parse version '%s': %w", from, err)
```

Comment:
Should be `to` here :smile: 

```suggestion
		return false, fmt.Errorf("failed to parse version '%s': %w", to, err)
```

Reply:
copy/paste too fast :no_mouth: